tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2052,574 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
+
+})
+
+var _ = Describe("{StoPoolExpMulPools}", func() {
+	var testrailID = 51298
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51298
+	// testrailID Description : Having multiple pools and resize only one pool
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("StoPoolExpMulPools", "Validate storage pool expansion using resize-disk option when multiple pools are present on the cluster", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+
+	It("Has to schedule apps, and expand it by resizing a pool", func() {
+
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("poolexpand-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		// Get all the storage Nodes present in the system
+		stNodes := node.GetStorageNodes()
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Storage nodes found!")
```

Comment:
I think this should say
```
dash.VerifyFatal(len(stNodes) > 0, true, "No storage nodes found!")
```
Since you have condition `len(stNodes) == 0`, correct me if I am wrong on this message logic